### DISCUSSION TITLE
能否加一个魔力换上传的插件。

### DISCUSSION THREAD

**Discussion by oynt** [jxxghp/MoviePilot#3430] [讨论]
### 当前程序版本

无

### 运行环境

Docker

### 功能改进类型

插件

### 功能改进

能否搞个自动化的魔力换上传插件。对于像我这样的佛系用户来说，并不经常性刷流，再加上国内的网络环境，上传实际上是一塌糊涂的。但是我保种做的不错，1是我看了不删会一直保种，2是我进了几个契约保种的站，经过辅种后单站保种数基本在50-300之间，体积在200G-3T之间，所以我的做种魔力还是不少的。
希望能出个魔力自动换上传插件，可以设定分享率低于多少自动换上传/魔力积累到多少就自动换上传等等。防止因为分享率低导致账号被封禁。
感谢。

**Comment by hoey94** [comment]
这个需要考虑的因素挺多的。
1. 每个PT站换积分换上传的方式不一样。
2. 每个PT站换积分的限制不一样，比如等级限制。
3. 每个PT站是否对外暴露接口，需要带验证。
